tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
+    state=present
+  tags:
+    - exim
+
+- name: "Make sure exim (exim4) is running"
+  service: >
+    name=exim4
+    state=started
+    enabled=yes
+  tags:
+    - exim
+
+
+- name: "Configure the package"
+  shell: |
+    executable=/bin/bash
+    dpkg-reconfigure -f noninteractive exim4-config &&\
+    update-exim4.conf
+  tags:
+    - exim
+
+- name: "Copy the template configuration"
+  template: >
+    src=update-exim4.conf
+    dest=/etc/exim4/update-exim4.conf
+    owner=root
+    group=root
+    mode=0644
+  tags:
```

Comment:
Hmm, ok, it's more or less connected to [the comments above](https://github.com/AerisCloud/ansible-exim/pull/1/files#r34329197) actually.